state-chain/cf-integration-tests/src/swapping.rs

```diff
@@ -401,12 +400,22 @@ fn can_process_ccm_via_swap_deposit_address() {
 				pallet_cf_ingress_egress::ChannelIdCounter::<Runtime, EthereumInstance>::get(),
 			)
 			.unwrap();
+		let ingress_fee = sp_std::cmp::min(
+			Swapping::calculate_input_for_gas_output::<Ethereum>(
+				cf_primitives::chains::assets::eth::Asset::Flip,
```

Comment:
nit: prefer to import to make it more readble. For example `use cf_primitives::chains::assets::eth::Asset as EthAsset`.